tests/servicebus_functions/servicebus_trigger_return/function.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
This is the actual trigger function.

Reply:
@pragnagopa ^

Reply:
From the documentation https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-service-bus looks like we only have OutputBinding and ServiceBusTrigger.

Reply:
Sorry, I don't understand.  This is a "serviceBusTrigger" function.  It _does_ get triggered.  My issue is that the data passed by the host to the tigger is incorrect.

Reply:
How is the function triggerred? Are you using storage sdk to input values into ServiceBusQueue?

Reply:
I put a message into a queue by calling another function with an output binding.

Reply:
Can you add link to that test here?

Reply:
It's in the last file in this PR.

`function.json`:

```json
{
  "scriptFile": "__init__.py",
  "disabled": false,

  "bindings": [
    {
      "type": "httpTrigger",
      "direction": "in",
      "name": "req"
    },
    {
      "direction": "out",
      "name": "msg",
      "queueName": "testqueue",
      "connection": "AzureWebJobsServiceBusConnectionString",
      "type": "serviceBus"
    },
    {
      "direction": "out",
      "name": "$return",
      "type": "http"
    }
  ]
}
```

`init.py`:

```python
import azure.functions as azf


def main(req: azf.HttpRequest, msg: azf.Out[str]):
    msg.set(req.get_body())

    return 'OK'
```

```